packages/fxa-auth-server/fxa-oauth-server/docs/api.md

```diff
@@ -74,8 +74,10 @@ The currently-defined error responses are:
   - [POST /v1/developer/activate][developer-activate]
 - [POST /v1/verify][verify]
 - [POST /v1/key-data][key-data]
-- [GET /v1/client-tokens][client-tokens]
-- [DELETE /v1/client-tokens/:id][client-tokens-delete]
+- [POST /v1/authorized-clients][authorized-clients]
+- [POST /v1/authorized-clients/destroy][authorized-clients-destroy]
+- (**DEPRECATED**) [GET /v1/client-tokens][client-tokens]
+- (**DEPRECATED**) [DELETE /v1/client-tokens/:id][client-tokens-delete]
```

Comment:
Out of curiosity, what is the versioning in the API being used for? Are there plans for a /v2/, or are we mainly claiming by versioning that we won't change individual route parameters/responses?

Reply:
I believe it's "just in case" versioning if we decide we want to make a wildly different API, but we don't have concrete plans for a `v2`. (And it's amazing how much time can otherwise be spent on bikeshedding versioning schemes for hypothetical future API changes).